### ISSUE TITLE
CesiumCreditSystem.cpp crash during runtime

### ISSUE THREAD

**Issue by argallegos** [CesiumGS/cesium-unreal#921]
From https://community.cesium.com/t/crashes-possibly-due-to-plugin-tile-loading-ue5/19641, reported by 2 users

Users are experiencing a crash while in play mode that produces the following error log.

```
Unhandled Exception: EXCEPTION_ACCESS_VIOLATION 0x0000000000000000

UnrealEditor_CesiumRuntime!ACesiumCreditSystem::Tick() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\CesiumForUnreal\Source\CesiumRuntime\Private\CesiumCreditSystem.cpp:210]
UnrealEditor_Engine
```
The crash doesn't seem to happen every time, but fairly regularly.

**Comment by argallegos** [comment]
Changes were made to the credit system fairly recently (https://github.com/CesiumGS/cesium-unreal/pull/868). It might be worth looking at what changed to see if anything unwanted was introduced.

**Comment by kring** [comment]
@joseph-kaile can you please take a look at this when you get a chance? If we need to fix something, it would be good to do that before the August release.

**Comment by joseph-kaile** [comment]
Yes, I will start working on that right away.

**Comment by nithinp7** [comment]
@argallegos This is most likely fixed now after #922 so I'm closing this, but if you or anyone on the community continues to experience the issue (after the August 1st release) feel free to reopen.
